spec/models/item_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Item, :type => :model do
+  let (:order) { Order.create(users_id: 1, status:"ordered", total: 1700)}
+  let (:item) { Item.create(title: "Green Chili Burrito", description:"Amazingly spicey burrito", price:655, image: "image/path")}
+
+  it 'is valid' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.